modules/ROOT/pages/install-workflow.adoc

```diff
@@ -13,6 +13,10 @@ The installer is interactive and walks you through a set of wizard pages that en
 
 The CLI installer does not require human interaction using a web browser and is useful for automation.
 
+NOTE: The installation package includes everything required for Anypoint Private Cloud Edition installation, 
```

Comment:
Format as a note if it's a note. 
[NOTE]
blah blahblahblah

Reply:
OK - thank you!